src/JuMP.jl

```diff
@@ -1234,22 +1234,30 @@ Return the value of the attribute `attr` from the model's MOI backend.
 function MOI.get(model::Model, attr::MOI.AbstractModelAttribute)
     if !MOI.is_set_by_optimize(attr)
         return MOI.get(backend(model), attr)
-    elseif attr isa MOI.TerminationStatus
-        if model.is_model_dirty && mode(model) != DIRECT
-            return MOI.OPTIMIZE_NOT_CALLED
-        end
-        return MOI.get(backend(model), attr)
-    elseif attr isa MOI.PrimalStatus || attr isa MOI.DualStatus
-        if model.is_model_dirty && mode(model) != DIRECT
-            return MOI.NO_SOLUTION
-        end
-        return MOI.get(backend(model), attr)
-    else
-        if model.is_model_dirty && mode(model) != DIRECT
-            throw(OptimizeNotCalled())
-        end
-        return _moi_get_result(backend(model), attr)
+    elseif model.is_model_dirty && mode(model) != DIRECT
+        @warn(
+            "The model has been modified since the last call to `optimize!`. " *
```

Comment:
is it possible that optimize was never called? if so, the message would be slightly missleading

Reply:
Yes, this will also get called if you as for `value(x)` before calling `optimize!`. Should we amend to 
```
The model has been modified since the last call to `optimize!` (or `optimize!` has not been called yet).
```

Reply:
good! +1 for the improved text.